charms/kfp-profile-controller/src/components/pebble_components.py

```diff
@@ -0,0 +1,65 @@
+import dataclasses
+import logging
+
+from charmed_kubeflow_chisme.components.pebble_component import PebbleServiceComponent
+from ops.pebble import Layer
+
+logger = logging.getLogger(__name__)
+
+
+@dataclasses.dataclass
+class KfpProfileControllerInputs:
+    """Defines the required inputs for KfpProfileControllerPebbleService."""
+
+    MINIO_SECRET: str
+    MINIO_HOST: str
+    MINIO_NAMESPACE: str
+    MINIO_PORT: str
+    MINIO_ACCESS_KEY: str
+    MINIO_SECRET_KEY: str
+    KFP_VERSION: str
+    KFP_DEFAULT_PIPELINE_ROOT: str
+    DISABLE_ISTIO_SIDECAR: str
+    CONTROLLER_PORT: int
+    METADATA_GRPC_SERVICE_HOST: str
+    METADATA_GRPC_SERVICE_PORT: str
+
+
+class KfpProfileControllerPebbleService(PebbleServiceComponent):
+    def get_layer(self) -> Layer:
+        """Pebble configuration layer for kfp-profile-controller."""
+        # TODO: Make this built-in, use the service/container names
+        logger.info("MlPipelineUiPebbleService.get_layer executing")
```

Comment:
Either update to your Component's name, or remove.  This really was just a debugging thing I had a while back, sorry!  We get logging from other parts of the setup that probably makes this redundant now
```suggestion
```